scripts/profiler/plot_nordic.py

```diff
@@ -558,18 +553,19 @@ def plot_events_from_file(
             color='r',
             markersize=self.draw_state.event_submit_markersize)
 
-        if self.processed_events.tracking_execution:
-            rects = []
-            for ev in self.processed_events.tracked_events:
-                rects.append(
-                    matplotlib.patches.Rectangle(
-                        (ev.proc_start_time,
-                         ev.submit.type_id - self.draw_state.event_processing_rect_height/2),
-                         ev.proc_end_time - ev.proc_start_time,
-                         self.draw_state.event_processing_rect_height,
-                         edgecolor='black'))
-
-            self.draw_state.ax.add_collection(PatchCollection(rects))
+        rects = []
+        for ev in self.processed_events.tracked_events:
+            if ev.proc_start_time is None:
```

Comment:
Add assert related to end_time too, please

Reply:
We have to assume that transmitted bytes are correct and thus tracked event has always both proc start and end time set to some value or both set to None.